coalib/parsing/DefaultArgParser.py

```diff
@@ -241,6 +241,15 @@ def default_arg_parser(formatter_class=None):
         '-r', '--relpath', nargs='?', const=True,
         help='return relative paths for files (must be called with --json)')
 
+    outputs_group.add_argument(
+        '--debug-bears', nargs='?', const=True,
+        help='Enable bear debugging with pdb, that can help to identify and'
+        'correct errors in bear code. Steps into bear code as soon as being'
```

Comment:
have a look at this on the cli. Terrible :)

Reply:
```
  --debug-bears         Debugging Bears willhelp user in identifying and
                        removing errors from Bear codewith the help of Pdb
                        interface, additionally user can passbear name as a
                        additional argument to debug those specific bear,if no
                        additional arguments passed then it will debug all the
                        bearspassed to a --bears or -b argument
```

Its not fine? I know there are some spacing problem.

Reply:
@jayvdb  this is what I am getting in my terminal.

Reply:
were you talking about spacing problem?